styles/styles.css

```
@@ -0,0 +1,652 @@
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root {
+    --purple: #5722CD;
+    --blue: #4845E4;
+    --pink: #ED4385;
+    --black: #0C0C0C;
+    --light-purple: #F2F0FF;
+    --light-grey: #F3F6F9;
+    --grey: #7A7A7A;
+    --white: #FFFFFF;
+}
+
+body {
+    font-family: Roboto, Arial, Helvetica, sans-serif;
+    font-weight: 300;
+    font-size: 20px;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+h1, h2, h3, h4, h5 {
+    font-family: Merriweather, 'Times New Roman', serif;
+}
+
+h1 {
+    font-size: 36px;
+    padding: 16px 0;
+    margin-bottom: 10px;
+
+}
+
+h2 {
+    font-size: 36px;
+    padding: 16px 0;
+}
+
+h3 {
+    font-size: 24px;
+    margin-bottom: 15px;
+}
+
+h4 {
+    font-size: 22px;
+    margin: 6px 0;
+    font-weight: 200;
+}
+
+h5 {
+    font-size: 26px;
+    padding: 12px 0;
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+/* --------------- 3.2 Other --------------------------- */
+
+img {
+    width: 100%;
+}
+
+a,
+a:visited,
+a:link {
+    text-decoration: none;
+    color: var(--purple);
+}
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header Section ------------------------- */
+
+.header-outer-container {
+    background: linear-gradient(to bottom right, var(--blue), var(--purple));
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    justify-content: center;
+    position: relative;
+}
+
+/* Ik had onderstaande voor de achtergrond ruis op de header. */
+/* Dat werkt wel, alleen werken dan de links niet meer. */
+/* Even geen idee hoe dit op te lossen. */
+
+/*.header-outer-container::before {*/
+/*    content: "";*/
+/*    position: absolute;*/
+/*    top: 0;*/
+/*    left: 0;*/
+/*    width: 100%;*/
+/*    height: 100%;*/
+/*    background-image: url(/assets/noise-texture.png);*/
+/*    opacity: 0.6;*/
+/*    z-index: 1;*/
+/*}*/
+
+.header-outer-container::after {
+    position: absolute;
+    content: "";
+    bottom: 0;
+    left: 0;
+    width: 0;
+    height: 0;
+    border-right: 100vw solid transparent;
+    border-bottom: 2vw solid var(--white);
+    z-index: 1;
+}
+
+.header-inner-container {
+    color: var(--white);
+    flex-basis: 1400px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    flex-shrink: 1;
+    flex-grow: 0;
+    margin: 0 20px;
+    padding: 5px 5px;
```

Dit zou je nog wat korter kunnen schrijven:
```suggestion
    padding: 5px;
```

---

Aangepast